.github/workflows/coverage-check.yml

```diff
@@ -22,13 +22,41 @@ jobs:
         uses: onbjerg/foundry-toolchain@v1
         with:
           version: nightly
+      
+      - name: Install dependencies
+        run: sudo apt-get install lcov
 
       - name: Get development branch coverage
         id: coverage-development
         run: |
           cd packages/contracts
+          
+          # generates lcov.info
+          forge coverage --report lcov
+
+          # Foundry uses relative paths but Hardhat uses absolute paths.
+          # Convert absolute paths to relative paths for consistency.
+          sed -i -e 's/\/.*solidity.//g' lcov.info
+
+          # Merge lcov files
+          lcov \
+              --rc lcov_branch_coverage=1 \
+              --add-tracefile lcov.info \
+              --output-file merged-lcov.info
+
+          # Filter out node_modules, test, and mock files
+          lcov \
+              --rc lcov_branch_coverage=1 \
+              --remove merged-lcov.info \
+              --output-file filtered-lcov.info \
+              "*node_modules*" "*test*" "*mock*" "*scriptscripting*"
+
+
+          # Generate summary
+          COVERAGE_DEVELOPMENT_OUTPUT=$(lcov \
+              --rc lcov_branch_coverage=1 \
+              --list filtered-lcov.info)
 
-          COVERAGE_DEVELOPMENT_OUTPUT=$(forge coverage)
           echo COVERAGE=$(echo "${COVERAGE_DEVELOPMENT_OUTPUT}" | tail -n 1 | awk '{print $4}' | tr -d '%') >> $GITHUB_OUTPUT
```

Comment:
Check the "Print coverages" step in this run https://github.com/ubiquity/ubiquity-dollar/actions/runs/5006049672/jobs/8970812635#step:8:1:
```
Development branch coverage: 362
PR branch coverage: 362
```

Seems like the values are wrong. Should they be `36.2` instead of `362` ?

Reply:
Should be fixed now!